Instruction on what to do if not using Cloud Shell

andynava opened the pull request:
Match instructions in https://docs.microsoft.com/en-us/azure/event-grid/resize-images-on-storage-blob-upload-event

azurecla commented:
Hi __@roopalik__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
Please resolve the merge conflict. Thanks.

rmca14 commented:
If you need guidance about how to resolve a local merge conflict, see http://sharepoint/sites/azurecontentguidance/wiki/Pages/Resolve%20a%20local%20merge%20conflict%20yourself.aspx

rmca14 commented:
Closed due to inactivity.

andynava committed:
Instruction on what to do if not using Cloud Shell

Match instructions in https://docs.microsoft.com/en-us/azure/event-grid/resize-images-on-storage-blob-upload-event

andynava committed:
Update storage-blob-event-quickstart.md
